.github/workflows/api-dotnetcore.yml

```diff
@@ -96,66 +91,3 @@ jobs:
           env_vars: C#
           # Specify whether or not CI build should fail if Codecov runs into an error during upload
           fail_ci_if_error: true
-
-      - name: SonarScanner for .NET 6 with pull request decoration support
```

Comment:
Removed - The results of this scan are used to generate a comment in the pull request thread.